libraries/AP_Quicktune/AP_Quicktune.cpp

```diff
@@ -0,0 +1,690 @@
+/*
+  C++ implementation of quicktune based on original lua script
+ */
+
+// quicktune is not performance sensitive, save flash
+#pragma GCC optimize("Os")
```

Comment:
We typically don't do this on a per library level. This make the flash cost not comparable with other features.

Reply:
we should be doing this in a lot of libraries. We should not be paying for extra flash cost in libraries that are not timing sensitive
I'm not doing this to "make quicktune look small", I'm doing it so we can fit more features in on quadplanes. I'd like to go through some other libraries and do the same

Reply:
> I'm not doing this to "make quicktune look small",

I know this is not the goal, but it is a side affect. It is made harder because Qautotune is also removed in the same PR. I have no idea of the actual flash cost of quciktune. I guess it must cost much less that Qautotune, but we could apply Os there too of course....

> I'd like to go through some other libraries and do the same

If the optimisation change was broken out into a separate PR it would be much easier for the team to get a feel for how much flash is up for grabs if we were to apply this to other libs.